### DISCUSSION TITLE
Override Jhipster default implementation

### DISCUSSION THREAD

**Discussion by kalaiselvan-rks** [jhipster/generator-jhipster#25939] [Q&A] [answered]
I have started to use Jhipster + Quarkus to build my Java application, I am using the Jhispter default implemented JWT token generation and authentication.
In this process, I have been seen that the issuer is named to https://www.jhipster.tech , when I change it to my webpage url, I am getting unauthorized while validating incoming request ( i.e. token validation). I need to override this and also have to change the .pem key file used for token process. I couldn't found any blogs related to this in internet, so seeking for some advise on this.

**Comment by mraible** [comment, marked as answer]
Hello @kalaiselvan-rks,

Did you change the following properties in your `application.properties` file?

```
mp.jwt.verify.issuer=https://www.jhipster.tech
jhipster.security.authentication.jwt.issuer=https://www.jhipster.tech
```

If so, it's possible you need to create a new `.pem` file. Tagging @vishal423 since he's the primary maintainer of this blueprint.

**Reply by kalaiselvan-rks** [reply]
@mraible Thanks for the response. I already got the solution. But still wondered why its's been made as two different properties?
